src/predefinedComponents/components/UserModal/UserModal.styles.js

```diff
@@ -1,5 +1,5 @@
 import { StyleSheet, Platform } from 'react-native'
-import { ifIphoneX } from 'react-native-iphone-x-helper'
+import { ifIphoneX } from '../../../utils'
```

Comment:
Good job on the removing the dependency 👍

Reply:
Thanks